wallet/lib/pages/detailed_asset_view/widgets/pdf_placeholder.dart

```diff
@@ -9,32 +9,41 @@ class PdfPlaceHolder extends StatelessWidget {
   final String nftUrl;
   final String nftName;
   final String thumbnailUrl;
+  final bool showIcon;
 
   const PdfPlaceHolder({
     Key? key,
     required this.nftUrl,
     required this.nftName,
     required this.thumbnailUrl,
+    this.showIcon = true,
   }) : super(key: key);
 
   Widget getPdfThumbnailFromUrl() {
```

Comment:
Cannot we make one widget for all Thmbnails?

Reply:
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/115459890/204041337-d0f70267-93da-4790-aa14-335fbacea3d3.png">


<img width="1271" alt="image" src="https://user-images.githubusercontent.com/115459890/204041395-731b563a-c927-4754-b8bd-d3a1d9191109.png">


<img width="1267" alt="image" src="https://user-images.githubusercontent.com/115459890/204041469-88614d08-08a1-4f20-99ee-1a276ba3e8ab.png">

Reply:
they are pretty different,,